jbpm-wb-process-runtime/jbpm-wb-process-runtime-backend/src/test/java/org/jbpm/workbench/pr/backend/server/RemoteProcessImageServiceImplTest.java

```diff
@@ -177,10 +183,10 @@ public void testProcessInstanceImageCustomColors() {
         String activeNodeBorderColor = "#888886";
 
         service.getProcessInstanceDiagram("", "", null,
-                                          completeNodeColor, completeNodeBorderColor, activeNodeBorderColor);
+                                          completeNodeColor, completeNodeBorderColor, activeNodeBorderColor, "");
```

Comment:
@bxf12315 makes sense that testProcessInstanceImageCustomColors checks specific values for color configuration. Could you add a variable with  an specific error Node Color and check that value is used i the call. (Like it is done for the other colors)

Reply:
@bxf12315  that's it, thanks!